tests/components/cloud/test_tts.py

```diff
@@ -42,54 +54,138 @@ def test_schema() -> None:
     tts.PLATFORM_SCHEMA({"platform": "cloud"})
 
 
+@pytest.mark.parametrize(
+    ("engine_id", "platform_config"),
+    [
+        (
+            DOMAIN,
+            None,
+        ),
+        (
+            DOMAIN,
+            {
+                "platform": DOMAIN,
+                "service_name": "yaml",
+                "language": "fr-FR",
+                "gender": "female",
+            },
+        ),
+    ],
+)
 async def test_prefs_default_voice(
-    hass: HomeAssistant, cloud_with_prefs, cloud_prefs
+    hass: HomeAssistant,
+    cloud: MagicMock,
+    set_cloud_prefs: Callable[[dict[str, Any]], Coroutine[Any, Any, None]],
```

Comment:
```suggestion
    set_cloud_prefs: Callable[[dict[str, Any]], Awaitable[Any]],
```

Reply:
Or None, I keep forgetting which one is which

Reply:
There's no ambigouty about the return value. It will always be a coroutine.

Reply:
What's the difference between an `Awaitable` and a `Coroutine` then? I've been using them both interchangeable

Reply:
`Awaitable` can be used when the object is something that can be awaited, a coroutine, an `asyncio.Task`, an `asyncio.Event`, etc and we can't or don't want to be more precise.